concepts/symbols/about.md

```diff
@@ -0,0 +1,80 @@
+# About
+
+[Symbols][symbols] are named identifiers that can be used to refer to a value.
+Symbols are created through a symbol literal, which is by prefixing a name with a `:` character, e.g. `:foo`.
+They also allow for being written as strings, e.g. `:"foo"`, which allows for example spaces in the name.
+
+```ruby
+:foo # => :foo
+:"foo boo" # => :"foo boo"
+```
+
+Symbols are used in many places in the language, including as keys in hashes, to represent method names and variable names.
```

Comment:
Maybe this is where we expand on that where it is used as the identifiers for methods and local variables.

As an aside, it is interesting that constants, keywords, methods, and local variables all are the exception to the rule of "Everything is an object".